docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -40,12 +40,9 @@ This table lists all the possible parameters of a given resource together with t
 |----------|:-------------:|------|
 | **metadata.name** | Yes | Specifies the name of the CR. |
 | **spec.description** | No | Describes the connected Application.  |
-| **spec.accessLabel** | No | Labels the Application when an ApplicationMapping is created. |
 | **spec.labels** | No | Defines the labels of the Application. |
 | **spec.services** | No | Contains all services that the Application provides. |
 | **spec.services.id** | Yes | Identifies the service that the Application provides. |
-| **spec.services.identifier** | No | Provides an additional identifier of the ServiceClass. |
-| **spec.services.name** | No | Represents a unique name of the service used by the Service Catalog. |
```

Comment:
I removed that 'cause it references Service Catalog. Should we replace it with anything or is it all good?

Reply:
Changed to 
```
| **spec.services.identifier** | No | Represents an additional identifier unique in the Application scope. Allows the external system to provide its own identifier. |
| **spec.services.name** | No | Represents a unique name of the service. Used for proxying in Central Application Gateway. |
```